torch/utils/mkldnn.py

```diff
@@ -33,6 +33,49 @@ def forward(self, x):
         return y
 
 
+class MkldnnConv1d(torch.jit.ScriptModule):
```

Comment:
There seem to be a lot of copy paste with Conv2d.
Would it be possible to have a common subclass here to reduce code duplication?

Reply:
Yes, indeed. The logic here is mostly copy paste. I have write a base class `MkldnnConvNd`.